glue/config.py

```diff
@@ -511,7 +511,11 @@ def supported_classes(self):
 
     def get_handler_for(self, data_or_class):
         for translator in self:
-            if isinstance(data_or_class, translator.target_cls) or data_or_class is translator.target_cls:
+            if data_or_class is translator.target_cls or type(data_or_class) is translator.target_cls:
```

Comment:
In principle should we do the loop twice, once over exact matches then once with isinstance?